src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -488,18 +492,27 @@ static void mongoSubCountersUpdateCount
   const std::string&  db,
   const std::string&  collection,
   const std::string&  subId,
-  long long           count
+  long long           count,
+  long long           failsCounter
 )
 {
   orion::BSONObjBuilder  condition;
   orion::BSONObjBuilder  update;
   orion::BSONObjBuilder  countB;
+  orion::BSONObjBuilder  failsCounterB;
+
   std::string  err;
 
   condition.append("_id", orion::OID(subId));
   countB.append(CSUB_COUNT, count);
   update.append("$inc", countB.obj());
 
+  if (failsCounter > 0)
+  {
+  failsCounterB.append(CSUB_FAILSCOUNTER, failsCounter);
+  update.append("$inc", failsCounterB.obj());
+  }
+ 
```

Comment:
Indent fix:

```suggestion
  if (failsCounter > 0)
  {
    failsCounterB.append(CSUB_FAILSCOUNTER, failsCounter);
    update.append("$inc", failsCounterB.obj());
  }
 
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/6b68a8c9ded45dd2e30c033f4d5000cddfab19de